README.md

```diff
@@ -1,2 +1,17 @@
 # 2017-Hestia
-A home automation app.
+A home automation system.
+
+## General Description 
+This repository contains the clientside code of the **Hestia** home automation system. The system consists of a server which has to be set up at home and an app.
+The server is used to manage all home automation devices. The app, which is available as an apk for the Android framework,
+connects to the server and provides an interface to control the devices. 
+
+The current code represents a Minimally Viable Product (MVP), which supports the Philips Hue system. We will extend this system in the near future to add support for more plugins. 
+
+## Setup
+To install the app, all that is needed is the apk. The apk can be obtained by pulling the current release (0.3) and then compiling it using Android Studio. 
```

Comment:
I think you should say the "master" branch, because the release 0.3 branch will be deleted soon, as we did with the other release branches